argon2/src/error.rs

```diff
@@ -123,6 +123,8 @@ impl From<Error> for password_hash::Error {
 impl core::error::Error for Error {
     fn source(&self) -> Option<&(dyn core::error::Error + 'static)> {
         match self {
+            // TODO: remove afer `B64Error` will implement `core::error::Error`
+            #[cfg(feature = "std")]
```

Comment:
This is a consequence of yanking `base64ct` v1.7 (see https://github.com/RustCrypto/formats/issues/1684).

@tarcieri 
What are you thoughts on `base64ct`'s MSRV? Should we release v2 with the bump?